migrations/V103__Rename_Intake_Process_Statuses.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE system_intakes
+SET process_status = 'Initial development underway'
+WHERE process_status = 'The project is already in development';
+
+UPDATE system_intakes
+SET process_status = ''
+WHERE process_status = 'The project is already funded';
```

Comment:
I know the AC mentioned "removing" this status, but practically I'm wondering if it makes sense to just completely remove it from the DB now that I look at this change. It'd mean that users just see nothing entered for this.

I wonder if it'd make sense to keep it as a legacy entry that exists, but just can't be entered anymore? 🤔

Reply:
[Looking for clarification here](https://cmsgov.slack.com/archives/C0305U1014Y/p1647888197741369)

Reply:
Looks like we're good to go - ignore these comments 😁

Reply:
Yeah, that status shouldn't be able to be entered any more, since it's no longer present in `src/constants/enums/processStages.ts`.